MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,10 +1,170 @@
 import UIKit
 
 final class MovieQuizViewController: UIViewController {
-    // MARK: - Lifecycle
+    @IBOutlet private var imageView: UIImageView!
+    @IBOutlet private var textLabel: UILabel!
+    @IBOutlet private var counterLabel: UILabel!
+    
+    private var currentQuestionIndex: Int = 0
+    private var correctAnswers: Int = 0
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        showNewQuestion()
     }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        imageView.layer.cornerRadius = 20
+    }
+    
+    @IBAction func noButtonClicked(_ sender: UIButton) {
+        let givenAnswer = false
+        showAnswerResult(isCorrect: givenAnswer == currentQuestion.correctAnswer)
+    }
+    
+    @IBAction func yesButtonClicked(_ sender: UIButton) {
+        let givenAnswer = true
+        showAnswerResult(isCorrect: givenAnswer == currentQuestion.correctAnswer)
+    }
+    
+    private var currentQuestion:QuizQuestion {
+        get {
+            return questions[currentQuestionIndex]
+        }
```

Comment:
если set-етер опускается, то можно использовать короткий синтаксис:
```suggestion
        return questions[currentQuestionIndex] 
```

и даже опустить return 

```suggestion
       questions[currentQuestionIndex] 
```